Jellyfin/Core/Central.cs

```diff
@@ -15,5 +15,15 @@ public static class Central
     /// <summary>
     /// Gets the minimum supported Jellyfin server version supported on this client.
     /// </summary>
-    public static Version MinimumSupportedServerVersion { get; } = new(10, 11, 0);
+    public static Version MinimumSupportedServerVersion { get; } = new(10, 10, 0);
+
+    /// <summary>
+    /// Gets the minimum supported Jellyfin server version supported on this client.
+    /// </summary>
+    public static Version MinimumFutureSupportedServerVersion { get; } = new(10, 11, 0);
```

Comment:
I Suppose this property is temporary for just this release?

Reply:
yea